deliver/lib/deliver/app_screenshot.rb

```diff
@@ -207,8 +209,8 @@ def self.device_messages
           [2778, 1284]
         ],
         ScreenSize::IOS_61_MESSAGES => [
-          [828, 1792],
-          [1792, 828]
+          [1179, 2556],
+          [2556, 1179]
```

Comment:
Is this going to be a breaking change for anybody that is currently relaying on the iPhone XR screenshot? It seems like since this resolution is changing that `deliver` will error out that it can't find a device for it. Is that correct? 🤔

Reply:
Yes, deliver will error out. The App Store doesn't allow iPhone XR screenshots anymore. This is what happens when you try to manually upload an iPhone XR screenshot:
<img width="669" alt="Screenshot 2022-11-16 at 11 20 42" src="https://user-images.githubusercontent.com/1105813/202154948-68941059-b8b8-412c-97b4-639522c26630.png">